examples/etl/multiple_transform_example.py

```diff
@@ -32,18 +37,18 @@ def process(self, df: DataFrame) -> DataFrame:
 
 
 class NoopLoader(Loader):
-    def save(self, df: DataFrame) -> DataFrame:
-        df.write.format('noop').mode('overwrite').save()
-        return df
-
-
-print('ETL Orchestrator using multiple transformers')
-etl = (Orchestration
-       .extract_from(GuitarExtractor())
-       .transform_with(IntegerColumnTransformer('id'))
-       .transform_with(IntegerColumnTransformer('year'))
-       .load_into(NoopLoader())
-       .build())
+    def save(self, df: DataFrame) -> None:
+        df.write.format("noop").mode("overwrite").save()
+
```

Comment:
why not returning the df?

Reply:
https://github.com/atc-net/atc-dataplatform/pull/57#discussion_r780823779